react/src/components/molecules/Link/index.js

```diff
@@ -1,17 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import './style.css';
 
 import { Icon } from '../../../index';
 
 const Link = (props) => {
   const icon = props.icon ? (<Icon name={props.icon} svgWidth={13.2} svgHeight={13.2} />) : '';
+  const classes = classNames({
+    'js-clickable-link': true,
```

Comment:
i still think we should remove this, in general we should clean up unused class and also, currently Link classes is not the a knob in the story we should add that. But don't have to be in this pr